summingbird-online/src/test/scala/com/twitter/summingbird/online/PlannerSpec.scala

```diff
@@ -94,14 +119,16 @@ class PlannerSpec extends WordSpec {
       .sumByKey(store2)
 
     val planned = Try(OnlinePlan(tail))
-    val path = TopologyPlannerLaws.dumpGraph(tail)
+
 
     planned match {
-      case Success(graph) => assert(true == true)
+      case Success(graph) => {
+        assert(true == true)
+      }
       case Failure(error) =>
         val path = TopologyPlannerLaws.dumpGraph(tail)
         error.printStackTrace
-        println("Dumped failing graph to: " + path)
+        println("Dumped failing graph for ' Must be able to plan user supplied Job A ' to: " + path)
```

Comment:
You should be able to put the failure message in assert statement below and not have to supply the name of test explicitly.
assert(false, "message")